xbmc/LangInfo.h

```diff
@@ -76,9 +99,28 @@ class CLangInfo : public ISettingCallback
   const std::string& GetTimeZone() const;
 
   const std::string& GetRegionLocale() const;
-  const std::string GetLanguageLocale(bool twochar = false) const;
 
-  bool ForceUnicodeFont() const { return m_currentRegion->m_forceUnicodeFont; }
+  /*!
+  \brief Returns the full locale of the current language.
+  */
+  const std::string& GetLocale() const;
+
+  /*!
+   \brief Returns the two character ISO 639-1 language code of the current language.
+   */
+  const std::string& GetLanguageLocale() const;
+
+  /*!
+   \brief Returns the <ll>_<TT> locale of the current language.
+   */
+  const std::string& GetLanguageTerritoryLocale() const;
```

Comment:
mhmm...based on the comments....`GetLocale()` and `GetLanguageTerritoryLocale()` will return the same ID?

@tamland you're right. A locale implementation (see Java or C# for example) is both...language ID + country ID. In this case `GetLocale` returns the full ID ll_TT.
I would do it like this (at least here at work we do it so, because other frameworks did it before us ^^)
`GetLanguageCode` returns only the ISO 639-X language code
`GetTerritoryCode` or `GetCountryCode` returns only the ISO 3166 country code